s3fs/tests/test_s3fs.py

```diff
@@ -49,77 +48,81 @@
 c = test_bucket_name + '/tmp/test/c'
 d = test_bucket_name + '/tmp/test/d'
 py35 = sys.version_info[:2] == (3, 5)
+port = 5555
+endpoint_uri = 'http://127.0.0.1:%s/' % port
 
 
-@pytest.yield_fixture
-def s3():
+@pytest.fixture()
+def s3_base():
     # writable local S3 system
-    with moto.mock_s3():
-        from botocore.session import Session
-        session = Session()
-        client = session.create_client('s3')
-        client.create_bucket(Bucket=test_bucket_name, ACL='public-read')
-
-        client.create_bucket(
-            Bucket=versioned_bucket_name, ACL='public-read')
-        client.put_bucket_versioning(
-            Bucket=versioned_bucket_name,
-            VersioningConfiguration={
-                'Status': 'Enabled'
-            }
-        )
-
-        # initialize secure bucket
-        client.create_bucket(
-            Bucket=secure_bucket_name, ACL='public-read')
-        policy = json.dumps({
-            "Version": "2012-10-17",
-            "Id": "PutObjPolicy",
-            "Statement": [
-                {
-                    "Sid": "DenyUnEncryptedObjectUploads",
-                    "Effect": "Deny",
-                    "Principal": "*",
-                    "Action": "s3:PutObject",
-                    "Resource": "arn:aws:s3:::{bucket_name}/*".format(
-                        bucket_name=secure_bucket_name),
-                    "Condition": {
-                        "StringNotEquals": {
-                            "s3:x-amz-server-side-encryption": "aws:kms"
-                        }
+    import shlex
+    import subprocess
+
+    proc = subprocess.Popen(shlex.split("moto_server s3 -p %s" % port))
+
+    timeout = 5
```

Comment:
Given this timeout, i'd highly recommend using the latest dev version of moto to run this with.

Reply:
What has changed?